recipes/pyutilib/meta.yaml

```diff
@@ -97,7 +97,3 @@ about:
   license: BSD License
   summary: 'PyUtilib: A collection of Python utilities'
 
-extras:
-  recipe-maintainers:
-    - whart222
```

Comment:
I just pushed an update.

On Tue, Apr 12, 2016 at 5:39 PM, Filipe <notifications@github.com> wrote:

> In recipes/pyutilib/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/331#discussion_r59458984>
> :
>
> > @@ -97,7 +97,3 @@ about:
> >    license: BSD License
> >    summary: 'PyUtilib: A collection of Python utilities'
> >
> > -extras:
> > -  recipe-maintainers:
> > -    - whart222
>
> Can you re-add so we can see the failure. We do expect that block to be
> there!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/331/files/3cdbb10e87b8f33fb611b768c1e8ece26d9911f1..79e192f0ac0d17b9554f605598a57a1d206c4aaf#r59458984>
>